build/bump.php

```diff
@@ -39,271 +32,257 @@ function usage($command)
-	'dev_devel'       => $versionSubParts[2] . (!empty($versionParts[1]) ? '-' . $versionParts[1] : '') . (!empty($versionParts[2]) ? '-' . $versionParts[2] : ''),
-	'dev_status'      => $dev_status,
-	'reldate'         => date('j-F-Y'),
-	'reltime'         => date('H:i'),
-	'reltz'           => 'GMT',
-	'credate'         => date('Y-m-d'),
-	'credate_de'      => date('d.m.Y'),
-	'install_credate' => date('Y-m'),
-	'install_version' => $versionSubParts[0] . '.' . $versionSubParts[1] . '.' . $versionSubParts[2],
-];
+			$skip = false;
+			foreach ($excludeDirectories as $excludeDir) {
+				if (strpos($relativePath, $excludeDir) === 0) {
+					$skip = true;
+					break;
+				}
+			}
 
-// Prints version information.
-echo PHP_EOL;
-echo 'Version data:' . PHP_EOL;
-echo '- Main:' . PHP_TAB . PHP_TAB . PHP_TAB . PHP_TAB . $version['main'] . PHP_EOL;
-echo '- Release:' . PHP_TAB . PHP_TAB . PHP_TAB . $version['release'] . PHP_EOL;
-echo '- Full:' . PHP_TAB . PHP_TAB . PHP_TAB . PHP_TAB . $version['full'] .  PHP_EOL;
-echo '- Dev Level:' . PHP_TAB . PHP_TAB . PHP_TAB . $version['dev_devel'] . PHP_EOL;
-echo '- Dev Status:' . PHP_TAB . PHP_TAB . PHP_TAB . $version['dev_status'] . PHP_EOL;
-echo '- Release date:' . PHP_TAB . PHP_TAB . PHP_TAB . $version['reldate'] . PHP_EOL;
-echo '- Release time:' . PHP_TAB . PHP_TAB . PHP_TAB . $version['reltime'] . PHP_EOL;
-echo '- Release timezone:' . PHP_TAB . PHP_TAB . $version['reltz'] . PHP_EOL;
-echo '- Creation date:' . PHP_TAB . PHP_TAB . $version['credate'] . PHP_EOL;
-echo '- Creation date DE:' . PHP_TAB . PHP_TAB . $version['credate_de'] . PHP_EOL;
-echo '- Installer: creation date:' . PHP_TAB . $version['install_credate'] . PHP_EOL;
-echo '- Installer: version:' . PHP_TAB . PHP_TAB . $version['install_version'] . PHP_EOL;
-
-
-echo PHP_EOL;
+			if ($skip) {
+				continue;
+			}
 
-$rootPath = dirname(__DIR__);
+			// Load file contents and replace copyright year
+			$contents = file_get_contents($filePath);
+			$updated = false;
 
-// Updates the version and creation date in language xml files.
-foreach ($languageXmlFiles as $languageXmlFile)
-{
-	if (file_exists($rootPath . $languageXmlFile))
-	{
-		$fileContents = file_get_contents($rootPath . $languageXmlFile);
-		$fileContents = preg_replace('#<version>[^<]*</version>#', '<version>' . $version['full'] . '</version>', $fileContents);
-		$fileContents = preg_replace('#<creationDate>[^<]*</creationDate>#', '<creationDate>' . $version['credate'] . '</creationDate>', $fileContents);
-		$fileContents = preg_replace('#<span class=\"jgerman-version\">(.*)<\/span>#', '<span class="jgerman-version">' . $version['main'] . '</span>', $fileContents);
-		file_put_contents($rootPath . $languageXmlFile, $fileContents);
+			if (preg_match('#2005\s+-\s+[0-9]{4}\s+Open\s+Source\s+Matters#', $contents)) {
```

Comment:
I would remove that check as the core does not update the the file headers right?

For example there we use: `; (C) 2020 Open Source Matters, Inc. <https://www.joomla.org>` for example

Reply:
see 6e4692da84b53d1adc2df021173980141f194685